samples/using-oidc-js-sdk/java-webapp/index.jsp

```diff
@@ -76,28 +81,24 @@
         </div>
     </body>
     <script src="https://cdn.jsdelivr.net/npm/axios@0.20.0/dist/axios.min.js"></script>
-    <script src="node_modules/@asgardio/oidc-js/dist/main.js"></script>
+    <!-- This script tag is only meant for development -->
+    <script src="assets/asgardio-oidc.production.min.js"></script>
```

Comment:
Do we need the `production` part in the artifact. If you have a `development` one you can keep it as `asgardio-oidc.development.min.js`, but skip specifying the environment in the production artifact. WDYT? 

cc: @jeradrutnam